docs/general/reverse-proxy.md

```diff
@@ -0,0 +1,24 @@
+# Reverse Proxy
```

Comment:
It would be helpful to have something in here covering the scary warning ("This App binds to the same ports as TrueNAS SCALE UI: 80 and 443") about ports 80/443 and what to do about it.

Reply:
Feel free to add ;-)

Reply:
Heh. I would if I had a clue what to do about it!

Reply:
it's a manual file, you open the file and write your explaination.

The help-text on the Traefik app is quite clear. Additionally the new version of traefik allows remapping ports for advanced users (which is NOT going to get support if used)

Reply:
No worries. I think I've figured it out, and I'll make some suggestions.

Reply:
Awesome, thanks! :)